pkg/util/mount/mount.go

```diff
@@ -143,6 +142,23 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, devPath string) ([]string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return nil, err
+	}
+	// Find all references to the device.
+	var refs []string
+	for i := range mps {
+		if mps[i].Path == devPath || mps[i].Device == devPath {
```

Comment:
Here is what I get from production: 
mps.Path: /mnt/disks/vol1 
mps.Device: /dev/vdb1 
globalPath: /mnt/disks/vol1
So I check for match between mps.Path and globalPath, but in case of Unit Testing, fake mounter reverses info "some global mount" goes into mps.Device. That is why for Unit testing I need '||' additional check in if.